guide.md

```diff
@@ -406,22 +406,44 @@ error 403 do
 end
 ```
 
-To use a custom `error` you must set the `response.status_code` and then raise a `Kemal::Exceptions::CustomException`.
+To handle a custom error based on a raised exception, you pass the exception to `error`
 
 ```ruby
 get "/" do |env|
   if some_condition
-    env.response.status_code = 403
-    raise Kemal::Exceptions::CustomException.new env
+    raise ValueError.new
   end
   {"message": "Hello Kemal"}.to_json
 end
 
-error 403 do
```

Comment:
Can we also keep this as an example code of using only `status_code`

Reply:
Wouldn't the status code handlers be covered by the example above? 

https://github.com/kemalcr/kemalcr.com/blob/91030d4c0f8996c53e534e003337160b7d06ff39/guide.md?plain=1#L400

Reply:
You're right 👍